cli/module_registry.go

```diff
@@ -679,3 +684,78 @@ func createTarballForUpload(moduleUploadPath string, stdout io.Writer) (string,
 	}
 	return tmpFile.Name(), nil
 }
+
+func readModels(path string, logger golog.Logger) ([]ModuleComponent, error) {
+	parentAddr, err := os.MkdirTemp("", "viam-cli-test-*")
+	if err != nil {
+		return nil, err
+	}
+	defer vutils.UncheckedErrorFunc(func() error { return os.RemoveAll(parentAddr) })
+	parentAddr += "/parent.sock"
+
+	cfg := modconfig.Module{
+		Name:    "xxxx",
+		ExePath: path,
+	}
+
+	mgr := modmanager.NewManager(parentAddr, logger, modmanageroptions.Options{UntrustedEnv: false})
```

Comment:
You should defer a close to this module manager.

Reply:
done